terra_notebook_utils/drs.py

```diff
@@ -153,8 +153,13 @@ def head(drs_url: str,
                                      f'{traceback.format_exc()}')
     blob = client.bucket(info.bucket_name, user_project=google_billing_project).blob(info.key)
     try:
+        # sys.stdout.buffer is used outside of a python notebook
+        # INSIDE of a python notebook, this is a ipykernel.iostream.OutStream object:
+        # https://github.com/ipython/ipykernel/blob/master/ipykernel/iostream.py#L265
+        # this seems to be able to handle bytes as well as unicode
+        stdout_buffer = getattr(sys.stdout, 'buffer', sys.stdout)
```

Comment:
Is this object guaranteed to have a `write` method?

Reply:
Yes.